src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -29,18 +29,22 @@ local tLines = {}
 local bRunning = true
 
 -- Colours
-local highlightColour, keywordColour, commentColour, textColour, bgColour, stringColour
+local highlightColour, booleanColour, keywordColour, functionColour, commentColour, textColour, bgColour, stringColour
 if term.isColour() then
-    bgColour = colours.black
-    textColour = colours.white
-    highlightColour = colours.yellow
-    keywordColour = colours.yellow
-    commentColour = colours.green
-    stringColour = colours.red
+    bgColour = settings.get("edit.backgroundColor") or colours.black
```

Comment:
- It's probably defining these settings at the bottom of `bios.lua` too.
 - Maybe make them strings instead? So one can define it as "red" instead.

Reply:
Good idea. I’ll look into that.